Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -25,15 +25,17 @@ struct DashboardView: View {
             List {
                 ForEach(viewModel.searchedEvents) { event in
                     VStack(alignment: .leading, spacing: 8) {
-                        Text(event.title)
+                      Text(event.title)
                             .font(.title3)
                         
-                        Text("For \(event.vehicle.name)")
+                        Text("For \(event.vehicle.name)",
+                             comment: "Maintenance list item description 'For which vehicle'")
                         
-                        Text("\(event.date.formatted(date: .abbreviated, time: .omitted))")
+                        Text("\(event.date.formatted(date: .abbreviated, time: .omitted))",
+                             comment: "Maintenance list item description 'Date' formatted")
                         
                         if !event.notes.isEmpty {
-                            Text(event.notes)
+                          Text(event.notes)
```

Comment:
This one isn't commented

Reply:
I have to doublecheck this, when I am on my laptop, but I might need to change the line to be Text("\(event.notes)", comment: ...) to be able to add the comment. That is ok, if you want to have the comment. However, there isn't anything to translate, so I am wondering if adding a comment makes any sense?

Reply:
That's a good point, probably don't need to translate it, no worries, I'm removing this